addons/main.go

```diff
@@ -255,4 +264,26 @@ func enableClusterBootstrapAndConfigControllers(ctx context.Context, mgr ctrl.Ma
 		setupLog.Error(err, "unable to create controller", "controller", "clusterbootstrap")
 		os.Exit(1)
 	}
+
+	setupCNIWebhooks(ctx, mgr, flags)
+}
+
+func setupCNIWebhooks(ctx context.Context, mgr ctrl.Manager, flags *addonFlags) {
+	certPath := path.Join(constants.WebhookCertDir, "tls.crt")
+	keyPath := path.Join(constants.WebhookCertDir, "tls.key")
+	if _, err := webhooks.InstallNewCertificates(ctx, mgr.GetConfig(), certPath, keyPath, constants.WebhookScrtName, flags.addonNamespace, constants.WebhookServiceName, "webhook-cert="+constants.CNIWebhookLabel); err != nil {
```

Comment:
lets standardize on the label and make it the same

Reply:
Standardized and Fixed